src/App.tsx

```diff
@@ -17,18 +52,33 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                filterBy={filterBy}
+                setFilterBy={setFilterBy}
+                query={query}
+                setQuery={setQuery}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              { !allTodos.length
+                ? (<Loader />)
```

Comment:
create `isLoading` state and set value when todos are loaded 

<img width="482" alt="Screenshot 2022-10-02 at 19 20 18" src="https://user-images.githubusercontent.com/47606379/193464823-d7f4b1bc-1d2d-4577-a582-ea7a85970c19.png">